tests/integration/helpers/is-mobile-test.ts

```diff
@@ -0,0 +1,23 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+import { setBreakpoint } from 'ember-responsive/test-support';
+
+module('Integration | Helper | is-mobile', function(hooks) {
+    setupRenderingTest(hooks);
+
+    // Replace this with your real tests.
+    test('it renders', async function(assert) {
+        setBreakpoint('mobile');
+
+        await render(hbs`
+<div data-test-div>{{if (is-mobile) 'mobile' 'not-mobile'}}</div>`);
```

Comment:
Question: the style looks off for me but I am not familiar either. Maybe something like this?

```javascript
await render(hbs`
        <div data-test-div>{{if (is-mobile) 'mobile' 'not-mobile'}}</div>
`);
```

Reply:
Yea, our linter doesn't like or know how to handle the `render(hbs` we use here. It always gives an "Incorrect indentation" message and says the indentation should be 0, for whatever reason. I like how your suggestion is styled much more, so I will disable the linter for this bit and update the indentation level